tests/env/platform/ports.go

```diff
@@ -32,12 +32,12 @@ const (
 	TestAddHeaders
```

Comment:
Can you add a comment on how you are sorting these automatically?

Reply:
I wish I could but it is done by IDE.